apps/cli/src/node/docker-compose-dev.yaml

```
@@ -183,15 +183,24 @@ services:
 
     prompt:
         image: debian:bookworm-slim
+        environment:
+            PROMPT_TXT_01_ANVIL: "Anvil running at http://localhost:8545"
+            PROMPT_TXT_02_GRAPHQL: "GraphQL running at http://localhost:8080/graphql"
+            PROMPT_TXT_03_INSPECT: "Inspect running at http://localhost:8080/inspect/"
+            PROMPT_TXT_04_EXPLORER: "Explorer running at http://localhost:8080/explorer/"
+            PROMPT_TXT_10_EXIT: "Press Ctrl+C to stop the node"
         command:
-            [
-                "sh",
-                "-c",
-                "ls | sort | xargs cat; trap exit INT TERM; sleep infinity & wait",
-            ]
+            - /bin/bash
+            - -c
+            - |
+                for txt_file in $${!PROMPT_TXT_*}
+                do
+                    declare -n txt_content=$$txt_file
+                    _txt_file=$${txt_file:11}
+                    echo "$$txt_content" > /prompt/$${_txt_file,,}.txt
+                done
+                ls | sort | xargs cat; trap exit INT TERM; sleep infinity & wait
```

Since we don't need to transform this into files, we could just print the messages.

```suggestion
                    echo "$$txt_file"
                done
```

Not sure if we need to `wait`.

If that's the case:

```suggestion
                    echo "$$txt_file"
                done
                trap exit INT TERM; sleep infinity & wait
```